src/lib/server/events/ical.test.ts

```diff
@@ -1,126 +0,0 @@
-import { test, expect, assert } from 'vitest';
-import { readFileSync, writeFileSync } from 'node:fs';
-import { Temporal } from '@js-temporal/polyfill';
-import { SAMPLE_EVENTS } from './data/sample-events';
-import {
-  makeEventSlug,
-  parseDescription,
-  makeEventLink,
-  wrapText,
-  walkICAL,
-  replaceHtmlLinkTargets,
-  parse,
-} from './ical';
-
-const ICAL_DATA = readFileSync('src/lib/server/events/data/sample-events.ics', 'utf-8');
-
-test('ICAL parser integration test', () => {
-  const events = parse(ICAL_DATA, { maxEvents: 10, filterBefore: false });
-
-  // To generate 'src/lib/server/events/data/sample-events.ts', uncomment the
-  // following lines of code then run 'npm t' (and 'npm run all'):
-  //
-  writeFileSync(
-    'src/lib/server/events/data/sample-events.ts',
-    `export const SAMPLE_EVENTS = ${JSON.stringify(events)};`
-  );
-
-  expect(events).toEqual(SAMPLE_EVENTS);
-});
```

Comment:
This end-to-end test no longer needs to be reimplemented. This is because the original `parse` function required an extra step to parse the ICAL data which we no longer require, so without this extra step we no longer need to test on any sample data.